### PR TITLE
enable / disable comments

### LINKED ISSUE #2286: Comments: Enable / Disable Comments
Based on discussion and agreements from https://github.com/Joystream/atlas/issues/2210

---

### User stories
1. As a channel owner, I want to enable or disable the comments section for my videos, separately.

**AC**
- [ ] Enable/ disable comments switch added to studio for video upload and editing. [Figma Link](https://www.figma.com/file/xm6vBdSnQDroc57rGfGtPM/Video-workspace?node-id=308%3A5753)
- [ ] Enable/ disable comments for minted video. [Figma>](https://www.figma.com/file/xm6vBdSnQDroc57rGfGtPM/Video-workspace?node-id=308%3A5753)
- [ ] Disabling the comments section does not allow for posting new comments under this video and hides any existing comments.

- [ ] Viewer side when comments were disabled > https://www.figma.com/file/sMydmZJlzhK91FTIPOd7D2/Video-page?node-id=2411%3A219280

### PR DESCRIPTION
**Pull request by rafalpawlow**
Closes #2286 

Because we don't have comments yet on QN (https://github.com/Joystream/joystream/pull/3340) I ignored AC related to integration

**Design**
https://www.figma.com/file/xm6vBdSnQDroc57rGfGtPM/Video-workspace?node-id=303%3A74

### CODE REVIEW

File: packages/atlas/src/views/studio/VideoWorkspace/VideoForm/VideoForm.tsx [hunk 1]
```diff
@@ -392,6 +392,40 @@ export const VideoForm: React.FC<VideoFormProps> = React.memo(({ onSubmit, setFo
...
+  const alwaysEditableFormFields = (
+    <ExtendedMarginFormField
+      title="Comments"
+      description="Disabling the comments section does not allow for posting new comments under this video and hides any existing comments made in the past."
+    >
+      <Controller
+        /* @ts-ignore TODO: this needs to be implemented on Query Node */
+        name="commentsEnabled"
+        control={control}
+        defaultValue="true"
```
Comment (drillprop) [inline packages/atlas/src/views/studio/VideoWorkspace/VideoForm/VideoForm.tsx]: We should set this as a boolean like we have in `isExplicit`.

**Review by drillprop — CHANGES_REQUESTED** [review]

**Review by drillprop — APPROVED** [review]
LGTM. Could you also add an issue for integration? I don't think we have that.

**Comment by drillprop** [comment]
Oh and don't forget to put in the issue:  

> Disabling the comments section does not allow for posting new comments under this video and hides any existing comments.
>  Viewer side when comments were disabled > https://www.figma.com/file/sMydmZJlzhK91FTIPOd7D2/Video-page?node-id=2411%3A219280

**Comment by rafalpawlow** [comment]
@drillprop https://github.com/Joystream/atlas/issues/2532

**Pull request merged**